static/app/components/projects/appStoreConnectContext/utils.tsx

```diff
@@ -18,27 +18,11 @@ export const appStoreConnectAlertMessage = {
 export function getAppConnectStoreUpdateAlertMessage(
   appConnectValidationData: AppStoreConnectValidationData
 ) {
-  if (appConnectValidationData.itunesSessionValid === false) {
+  if (appConnectValidationData.promptItunesSession === true) {
     return appStoreConnectAlertMessage.iTunesSessionInvalid;
   }
-
   if (appConnectValidationData.appstoreCredentialsValid === false) {
     return appStoreConnectAlertMessage.appStoreCredentialsInvalid;
   }
-
-  const itunesSessionRefreshAt = appConnectValidationData.itunesSessionRefreshAt;
-
-  if (!itunesSessionRefreshAt) {
-    return undefined;
-  }
-
-  const isTodayAfterItunesSessionRefreshAt = moment().isAfter(
-    moment(itunesSessionRefreshAt)
-  );
```

Comment:
since these are the only usages of `moment` in this file, it'd be best to remove its import at the top as well